src/main/java/com/arabiccalligraphy/api/DAO/ArtworkDAO.java

```diff
@@ -66,8 +66,17 @@ private void loadArtworkFromFile(String fileName) throws IOException {
             //Artist creation
             Artist artist = new Artist(contentItem.get("artist").asText());
 
+            //width creation
+            int width = contentItem.get("width").asInt();
+
+            //height creation
+            int height = contentItem.get("height").asInt();
+
+            //id creation
+            int id = contentItem.get("id").asInt();
+
             //artwork created
-            Artwork artwork = new Artwork(title, tags, image, artist);
+            Artwork artwork = new Artwork(title, tags, image, artist, width, height, id);
```

Comment:
This is a nitpick, but `id` is usually the first argument in the constructor.  I would re-organize the argument list to have `id` before `title`.